docs/logging-format.md

```diff
@@ -228,6 +228,45 @@ and TLS probe points.
 | `tls::key_exchange_algorithm` | uint16     | Key exchange mode: ECDHE(0), DHE(1), PSK(2), ECDHE-PSK(3), DHE-PSK(4)                            |
 | `tls::group`                  | uint16     | Groups used in the handshake (as in IANA [registry][iana-tls-supported-groups])                  |
 
+##### SSH context names
+
+| name                   | description                            |
+|------------------------|----------------------------------------|
+| `ssh::handshake_client`| SSH handshake for client               |
+| `ssh::handshake_server`| SSH handshake for server               |
+| `ssh::client_key`      | SSH client key signature/verification  |
+| `ssh::server_key`      | SSH server key signature/verification  |
+| `ssh::key_exchange`    | SSH key exchange                       |
+
+Example of SSH context tree:
+
+- `ssh::handshake_client`
+  - `ssh::ident_string` = `SSH-2.0-OpenSSH_8.8`
+  - `ssh::client key`
+  - `ssh::key_exchange`
+    - `ssh::kex_algorithm` = `curve25519-sha256`
+    - `ssh::c2s_cipher` = `aes256-gcm@openssh.com`
+
+##### SSH keys
+
+All the keys except `rsa_bits` have `string` type.
+We distinguish server and client values by the context we are in. We log all relevant events in both contexts.
+
+| key                             | description                                      | example                    |
+|---------------------------------|--------------------------------------------------|----------------------------|
+| `ssh::ident_string`             | Software identification string                   | `SSH-2.0-OpenSSH_8.8`      |
+| `ssh::key_algorithm`            | Key used in handshake                            | `ssh-ed25519`              |
```

Comment:
the description needs to be updated as if I understand the specification this should be also in the `client_key` and `server_key` and not (only) as part of the handshake.

Reply:
Updated, thanks for your feedback!